Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -11,32 +11,29 @@
  */
 'use strict';
 
-var EdgeInsetsPropType = require('EdgeInsetsPropType');
-var Platform = require('Platform');
-var PointPropType = require('PointPropType');
-var RCTScrollView = require('NativeModules').UIManager.RCTScrollView;
-var RCTScrollViewManager = require('NativeModules').ScrollViewManager;
-var React = require('React');
-var ReactNativeViewAttributes = require('ReactNativeViewAttributes');
-var ScrollResponder = require('ScrollResponder');
-var StyleSheet = require('StyleSheet');
-var StyleSheetPropType = require('StyleSheetPropType');
-var View = require('View');
-var ViewStylePropTypes = require('ViewStylePropTypes');
+const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const Platform = require('Platform');
+const PointPropType = require('PointPropType');
+const RCTScrollViewManager = require('NativeModules').ScrollViewManager;
+const React = require('React');
+const ScrollResponder = require('ScrollResponder');
+const StyleSheet = require('StyleSheet');
+const StyleSheetPropType = require('StyleSheetPropType');
+const View = require('View');
+const ViewStylePropTypes = require('ViewStylePropTypes');
 
-var deepDiffer = require('deepDiffer');
-var dismissKeyboard = require('dismissKeyboard');
-var flattenStyle = require('flattenStyle');
-var insetsDiffer = require('insetsDiffer');
-var invariant = require('invariant');
-var pointsDiffer = require('pointsDiffer');
-var requireNativeComponent = require('requireNativeComponent');
-var processColor = require('processColor');
+const dismissKeyboard = require('dismissKeyboard');
+const flattenStyle = require('flattenStyle');
+const invariant = require('invariant');
+const requireNativeComponent = require('requireNativeComponent');
+const processColor = require('processColor');
 
-var PropTypes = React.PropTypes;
+import type ReactComponent from 'ReactComponent';
```

Comment:
I just checked other files for 'import type' and it is always right under the requires.